addons/point_of_sale/static/src/js/models.js

```diff
@@ -2345,7 +2345,7 @@ exports.NumpadState = Backbone.Model.extend({
             this.set({
                 buffer: "-" + newChar
             });
-        } else {
+        } else if (!(newChar === '.') || oldBuffer.indexOf('.') === -1) {
```

Comment:
I don't really understand. It seems to me this will prevent entering a decimal dot at all.